src/test/scala/com/cognite/sdk/scala/common/GzipSttpBackendTest.scala

```diff
@@ -11,6 +11,7 @@ import java.util.zip.GZIPInputStream
 import cats.effect.IO
 import sttp.client3._
 import sttp.client3.asynchttpclient.cats.AsyncHttpClientCatsBackend
+
```

Comment:
We should move this file to `package com.cognite.sdk.scala.sttp` within test.

Reply:
Also, this should of course be renamed to `GzipBackendTest` to match the other rename.